### PR TITLE
New one

### PR DESCRIPTION
**Pull request by aurakanzaa**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -158,6 +159,7 @@
 + Safei Muslim: [@safeimuslim](https://medium.com/@safeimuslim)
 + Sastra Panca Nababan: [@sastranababan](https://medium.com/@sastranababan)
 + Satya Kresna Adi Pratama: [@satyakresna](https://medium.com/@satyakresna)
++ Sebastian Eschweiler: [@s_eschweiler](https://medium.com/@s_eschweiler)
```
Comment (mazipan) [inline README.md:162]: Delete this new line

**Review by mazipan — CHANGES_REQUESTED** [review]

File: README.md [hunk 2]
```diff
@@ -192,7 +193,7 @@
 
 + Yahya Fadhlulloh Al-Fatih: [@yahya.kimochi](https://medium.com/@yahya.kimochi)
 + Yudi Setiawan: [@kolonel.yudisetiawan](https://medium.com/@kolonel.yudisetiawan)
-
++ Yessika Budiono: [@yessika21289](https://medium.com/@yessika21289PW)
```
Comment (mazipan) [inline README.md:196]: This is 404

**Review by mazipan — CHANGES_REQUESTED** [review]

File: README.md [hunk 3]
```diff
@@ -113,6 +113,7 @@
 ## M
 
 + M Nafian Wildana: [@mnafian](https://medium.com/@mnafian)
++ Mohammad Azam [@azamsharp](https://medium.com/@azamsharp)
```
Comment (mazipan) [inline README.md:116]: This is not Indonesian

**Review by mazipan — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (aurakanzaa) [reply README.md]: omg, i hope i dont make another mistake again

**Review by mazipan — APPROVED** [review]
Looks good 🙂

**Pull request merged**